debian-pkg/debian/rules

```diff
@@ -24,6 +24,8 @@ override_dh_auto_install:
 	# Use system-provided files.
 	rm -fv 'build/src/python/docs/html/_static/jquery.js' 'build/src/python/docs/html/_static/underscore.js' \
 	       'build/docs/libcomps-doc/jquery.js'
+	# replace buildpath in xml
+	# find 'build/docs/libcomps-doc/xml' -name "*.xml" -exec sed -i "s|$(PWD)|.|g" {} \;
```

Comment:
Since you first add this and then disable, better simply drop commit adding this in the first place.

Reply:
I first thought to do that but here it was for keeping some previous attempt. I can use a separate branch but in case anyone would have solved this later it would seen previous fixes needed. For example, up to remaining node ordering, they is hardcoded build location into the xml. So this fix would be need when reenabling XML (if needed...).